src/support/variables/misc.scss

```diff
@@ -15,12 +15,21 @@ $box-shadow-extra-large: 0 10px 50px rgba($black, 0.07) !default;
 
 // Button and form variables
 $form-control-shadow: inset 0 1px 2px rgba($black, 0.075) !default;
-$btn-input-focus-shadow: 0 0 0 0.2em rgba($blue, 0.3) !default;
-$btn-active-shadow: inset 0 0.15em 0.3em $black-fade-15 !default;
+$btn-input-focus-shadow: 0 0 0 0.2em rgba($blue, 0.3) !default; // TODO: Deprecate? Replaced by $focus-shadow
+$btn-active-shadow: inset 0 0.15em 0.3em $black-fade-15 !default; // TODO: Deprecate? It's now unused.
+
+$focus-shadow: 0 0 0 3px rgba($blue, 0.4) !default;
```

Comment:
This is a new variable without the `btn` in the name that we might can use with inputs and other components.